bazel/emscripten_toolchain/toolchain.bzl

```diff
@@ -1040,6 +1040,10 @@ def _impl(ctx):
                     key = "EM_CONFIG_PATH",
                     value = ctx.file.em_config.path,
                 ),
+                env_entry(
```

Comment:
Why not just set `EMSDK_PYTHON` set here and avoid re-exporting it in env.sh?

Reply:
Done!